SofaKernel/modules/SofaBaseTopology/TopologyData.inl

```diff
@@ -57,7 +57,7 @@ void TopologyDataImpl <TopologyElementType, VecT>::createTopologicalEngine(sofa:
         this->m_topologicalEngine->setNamePrefix(std::string(sofa::core::topology::TopologyElementInfo<TopologyElementType>::name()) + std::string("Engine_"));
         if (this->getOwner() && dynamic_cast<sofa::core::objectmodel::BaseObject*>(this->getOwner())) dynamic_cast<sofa::core::objectmodel::BaseObject*>(this->getOwner())->addSlave(this->m_topologicalEngine.get());
         this->m_topologicalEngine->init();
-        this->linkToElementDataArray();
+        this->linkToElementDataArray((TopologyElementType*)NULL);
```

Comment:
Small understanding question: According to the changes done, if here `this` is a `PointData`, passing a NULL Pointer will still call `linkToPointDataArray` ?